tox.ini

```diff
@@ -45,9 +45,31 @@ setenv =
 whitelist_externals =
     npm
 
-[testenv:cypress]
+[testenv:cypress_dashboard]
```

Comment:
The use of [factor conditions](https://tox.readthedocs.io/en/latest/config.html#complex-factor-conditions) for the command could help reduce the repetition.